groups.go

```diff
@@ -508,6 +521,26 @@ func (s *GroupsService) DeleteGroupLDAPLink(gid interface{}, cn string, options
 	return s.client.Do(req, nil)
 }
 
+// DeleteGroupLDAPLinkWithCNOrFilter deletes a group LDAP link. Available only for users who
+// can edit groups.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/groups.html#delete-ldap-group-link-with-cn-or-filter
+func (s *GroupsService) DeleteGroupLDAPLinkWithCNOrFilter(gid interface{}, opts *DeleteGroupLDAPLinkWithCNOrFilterOptions, options ...RequestOptionFunc) (*Response, error) {
```

Comment:
I think calling this just `DeleteGroupLDAPLink` would probably make more sense, right?

Reply:
I tend to agree, but there already is such a function with a single string parameter for the `cn` field, so that would be a breaking change.

Reply:
Check... Then I'll merge this as is 👍🏻